### DISCUSSION TITLE
Scheduling GitHub App to run hourly

### DISCUSSION THREAD

**Discussion by dbasilio** [renovatebot/renovate#15453] [Request Help]
We recently made a few changes to our renovate config which has dramatically slowed down the pace of renovate PRs we're getting. Here are the relevant fields:
```json
{
    "schedule": "after 10am and before 4pm every weekday",
    "prCreation": "not-pending",
    "stabilityDays": 21,
    "prConcurrentLimit": 2,
    "internalChecksFilter": "strict"
}
```

We only run during our core hours because we have automerge turned on in a few places and want to guarantee those get deployed only when someone is around. We're currently seeing the renovate bot pass by our repos once or twice during those 6 hours though. Since we require checks to pass, that means we often go a day or more without seeing a PR put up because renovate ran at:
- 9:50am - did nothing
- noon - put up a branch to start running checks 
- 4:30pm - did nothing

We've seen this happening for a few weeks now, and monitoring the app run logs, it looks like Renovate only runs on a repo once every 3-5 hours.

I tried the below schedule to see if that would cause renovate to run more frequently but it looks like it's maintaining its existing schedule of running on a repo every few hours.
```json
{
    "schedule": "every 1 hour after 10am and before 4pm every weekday"
}
```

Is there a way to get it to run more frequently (especially during that 6 hour window)?

**Comment by rarkins** [comment]
The job queue is currently around 2.5x the hourly capacity. We are working on optimizing Renovate to take less time per repo. We may offer in future the ability of a paid plan in order to guarantee hourly SLAs if still needed. In the meantime you could potentially hack together a solution to tick the checkbox on your dependency dashboard hourly..

**Reply by dbasilio** [reply]
Sounds good, thanks for the quick reply!

**Reply by dbasilio** [reply]
Actually a follow up question, the reason our timeline is so restrictive is because of automerge, we could run renovate itself all day long if we could separate the schedule for merging from the schedule for running. Any chance that's been considered?
i.e.
```json
{
    "schedule": "after 6am and before 6pm",
    "automergeSchedule": "after 10am and before 4pm"
}
```

**Reply by rarkins** [reply]
Yes, been considered, but I don't think we have a feature request yet so please go ahead and create one if you wish

**Reply by dbasilio** [reply]
Will do 🙏
